ci/e2e-tests/test-run.sh

```diff
@@ -689,8 +727,9 @@ case "$OS" in
         ssh -i "$PWD/vm-ssh-key" "aziot@$vm_public_ip" '
```

Comment:
Didn't we need a retry around this whole too? Because the package repo would sometimes be borked and not find jq?

Reply:
Oh, just saw the commit message.